extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.tracking.js

```diff
@@ -0,0 +1,35 @@
+define('wikia.articleVideo.featuredVideo.tracking', [], function () {
+	function getDefaultState() {
+		return {
+			wasAlreadyUnmuted: false
```

Comment:
what do we need this flag for?

Reply:
This method will be used for resetting the tracking state before subsequent videos.